pennylane_qulacs/qulacs_device.py

```diff
@@ -39,8 +68,19 @@
         [0, 0, 0, np.exp(1j * theta / 2)],
     ]
 )
-toffoli = np.diag([1 for i in range(8)])
-toffoli[6:8, 6:8] = np.array([[0, 1], [1, 0]])
+
+
+def _reverse_state(state_vector):
+    """Reverse the qubit order for a vector of amplitudes.
+    Args:
+        state_vector (iterable[complex]): vector containing the amplitudes
+    Returns:
+        list[complex]
+    """
+    state_vector = np.array(state_vector)
+    N = int(np.log2(len(state_vector)))
+    reversed_state = state_vector.reshape([2] * N).T.flatten()
+    return list(reversed_state)
 
 
 def hermitian(*args):
```

Comment:
Do we actually need this for anything or should we simply remove this function?

Reply:
Let's remove! I'm sure PennyLane tests the Hermitian, and they are not supported anyways at this stage